tests/e2e/e2e_test.go

```diff
@@ -111,3 +111,76 @@ func (s *IntegrationTestSuite) TestBankTokenTransfer() {
 		)
 	})
 }
+
+/*
+bank send with fees two fees.
+The following test is for the case that:
+photon fee: higher than or equal to min_gas_price in app.toml, but lower than the photon fee required in globalfee.
+stake fee: higher than stake fee required in globalfee.
```

Comment:
So to clarify, as long as the transaction satisfies one of the global fee denoms, it will execute?

Reply:
no, fee has to be 
1. higher or equal to `min_gas_price` in app.toml && 2. higher or equal to one of the fee in the global fee list.